docs/docusaurus/src/components/WasThisHelpful/index.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import styles from './styles.module.css';
+import useBaseUrl from "@docusaurus/useBaseUrl";
+import {useLocation} from "@docusaurus/router";
+
+export default function WasThisHelpful(){
+
+    const thumbsUpImg = useBaseUrl(`img/thumbs_up_icon.svg`);
+    const thumbsDownImg = useBaseUrl(`img/thumbs_down_icon.svg`);
+
+    const { pathname } = useLocation();
+
+    const handleThumbsUp = () => {
+        posthog.capture('test_docs.thumbs_up', { doc_url: pathname })
+    };
+
+    const handleThumbsDown = () => {
+        posthog.capture('test_docs.thumbs_down', { doc_url: pathname });
+    };
+
+    return <>
+        <div className={styles.feedbackCard}>
+            <div className={styles.feedbackCardTitle}>
+                <h3>Was this helpful?</h3>
+            </div>
```

Comment:
We should be able to apply styling into the h3 directly. 
```suggestion
                <h3>Was this helpful?</h3>
```